File: java/Car.java
class Car {
    private Integer id;
    private String license;
    private Account driver;
    private Integer passenger;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    void printDataCar(){
        System.out.println("license: " + license + " Name Driver: " + driver.name + " Passengers: " + passenger)
    }

    public Integer getPassenger() {
        return passenger;
    }
    public void setPassenger(Integer passenger) {
        if(passenger == 4) {
            this.passenger = passenger;
        } else {
            System.out.println("Necesitas asignar 4 pasageros");
        }
    }
    
}

File: php/car.php
<?php

class Car {
  public $id;
  public $license;
  public $driver;
  protected $passenger; // Es importante que esta propiedad esté marcada como protected

  public function __construct($license, $driver) {
    $this->license = $license;
    $this->driver = $driver;
  }

  public function PrintDataCar(){
    echo "
      license: $this->license
      conductor: {$this->driver->name}
      document: {$this->driver->document}
      Número de pasajeros: $this->passenger
      ";
  }
  public function getPassenger() {
    return $this->passenger;
  }
  public function setPassenger($passenger) {     
    if ($passenger == 4) {
        $this->passenger = $passenger;
    }
    else {
        echo "Necesitas asignar 4 pasajeros";
    }
  }
}

?>